vllm/envs.py

```diff
@@ -585,6 +586,13 @@ def maybe_convert_int(value: Optional[str]) -> Optional[int]:
     # specify the path through environment variable VLLM_CUDART_SO_PATH.
     "VLLM_CUDART_SO_PATH":
     lambda: os.getenv("VLLM_CUDART_SO_PATH", None),
+
+    # Contiguous cache fetching to avoid using costly gather operation on
+    # Gaudi3. This is only applicable to HPU contiguous cache. If set to true,
+    # contiguous cache fetch will be used.
+    "VLLM_USE_HPU_CONTIGUOUS_CACHE_FETCH":
+    lambda: os.environ.get('VLLM_CONTIGUOUS_PA', 'true').lower(
+    ) in ['1', 'true'],
```

Comment:
Use tuple with double quotes.
```suggestion
    ) in ("1", "true"),
```